tests/config/macro-changed.cfg

```diff
@@ -0,0 +1,3 @@
+indent_with_tabs = 0
+macro-open  MACRO2_BEGIN_STUFF
```

Comment:
have no idea what is this?

Reply:
anybody to approve?

Reply:
ok but you haven't answered what is `macro-open` and where is it used?

Reply:
Sorry. macro-open and others are documentated at
option.cpp:73:
```.cpp
# - Custom macro-based indentation can be set up using 'macro-open',
#   'macro-else' and 'macro-close'.
#     `(macro-open | macro-else | macro-close) tokenString`
#
#     Example:
#       `macro-open  BEGIN_TEMPLATE_MESSAGE_MAP`
#       `macro-open  BEGIN_MESSAGE_MAP`
#       `macro-close END_MESSAGE_MAP`
```
and decoded at option.cpp:2299
```.cpp
   else if (strcasecmp(args[0], "macro-open") == 0)
   {
      add_keyword(args[1], CT_MACRO_OPEN);
   }
   else if (strcasecmp(args[0], "macro-close") == 0)
   {
      add_keyword(args[1], CT_MACRO_CLOSE);
   }
   else if (strcasecmp(args[0], "macro-else") == 0)
   {
      add_keyword(args[1], CT_MACRO_ELSE);
```